homeassistant/components/rest/sensor.py

```diff
@@ -197,13 +197,16 @@ def update(self):
             if value:
                 try:
                     json_dict = json.loads(value)
+                    if isinstance(json_dict, list):
+                        json_dict = json_dict[0]
```

Comment:
This requires a test.

Reply:
Ok, just committed.

Reply:
And the online tests just passed for my improved test (I had the test case wrong when I quickly did it in response to your request -- the actual code was right and had been tested but now the test case reflects that testing).  Thanks!